docs/hub/datasets-viewer-configure.md

```diff
@@ -0,0 +1,40 @@
+# Configure the Dataset Viewer
+
+The Dataset Viewer supports many data files formats, from text to tabular and from image to audio formats.
+It also separates the train/validation/test splits based on file and folder names.
+
+To configure the Dataset Viewer for your dataset, make sure your dataset is in a supported data format and structured the right way.
+
+## Supported data formats
+
+The dataset viewer supports multiple file formats:
+
+- CSV (.csv, .tsv)
+- JSON Lines, JSON (.jsonl, .json)
+- Parquet (.parquet)
+- Text (.txt)
+- Images (.png, .jpg, etc.)
+- Audio (.wav, .mp3, etc.)
+
+The dataset viewer also supports files compressed using ZIP (.zip), GZIP (.gz), ZSTD (.zst), BZ2 (.bz2), LZ4 (.lz4) and LZMA (.xz).
```

Comment:
Maybe give a hint here that we can have both CSV and images, for example. Or mention the ImageFolder format?